ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml

```diff
@@ -43,6 +43,21 @@ tests:
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: installer-rehearse-aws-disconnected-private
+  interval: 960h
+  steps:
+    cluster_profile: aws-qe
+    env:
+      ADD_INGRESS_RECORDS_MANUALLY: "yes"
```

Comment:
I think we can move the `ADD_INGRESS_RECORDS_MANUALLY` env to provision/deprovision chain.